server/models/Notification.js

```diff
@@ -3,22 +3,39 @@ const Schema = mongoose.Schema;
 
 const notificationSchema = new Schema({
     
-    receiver: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
-    notifications: [{
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User',
+        required: true,
+    },
+  
+    notifications: [
+        {
         type: {
             type: String,
-            enum: ['serviceRequest', 'serviceAvailable'], 
+            enum: ['serviceRequest', 'serviceAvailable'],
+            
         },
-        sender: {type:mongoose.Schema.Types.ObjectId, ref:'User'}, 
-        title: {type: String},
-        message: {type: String},
-        isRead: {type: Boolean,default: false},
-        created_at:{type: Date, default: Date.now},
-    }
-        
+        sender: {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'User',
+            required: true,
+        },
+            title: {
+                type: String
+            },
+            message: {
+                type: String
+            },
+            isRead: {
+                type: Boolean,
+                default: false
+            },
+            created_at: {
+                type: Date,
+                default: Date.now
+            },
```

Comment:
You can add mongoose `timestamps` option here to automatically manage `createdAt` and `updatedAt` properties.